### PR TITLE
Try fixing CTA test.

### PR DESCRIPTION
**Pull request by miina**
Improves the CTA test by explicitly selecting the Page block before Removing.

### CODE REVIEW

**Comment by miina** [comment]
Ran the e2e tests 3 times, passed every time.

**Comment by miina** [comment]
Done. Maybe it will fail now :D

**Comment by miina** [comment]
And yes, it did fail now. Will look into it tomorrow again then.

File: tests/e2e/specs/stories-editor/cta.test.js [hunk 1]
```diff
@@ -47,9 +48,7 @@ describe( 'Stories Editor Screen', () => {
 		expect( nodes ).toHaveLength( 1 );
 	} );
 
-	// Disable reason: Test is flaky in headless mode - needs to be investigated.
-	// eslint-disable-next-line jest/no-disabled-tests
-	it.skip( 'should display validation error when CTA block is on the first Page', async () => {
+	it( 'should display validation error when CTA block is on the first Page', async () => {
 		await createNewPost( { postType: 'amp_story' } );
 
 		const firstPageClientId = ( await getAllBlocks() )[ 0 ].clientId;
```
Comment (swissspidy) [inline tests/e2e/specs/stories-editor/cta.test.js]: We have lots of instances of `( await getAllBlocks() )[ 0 ].clientId`. I feel like we could use a `getPageClientId( pageNum )` utility function for that. And perhaps even a `getCurrentPageClient()` helper.

**Comment by miina** [comment]
OK, now it passed 5 times in a row.

**Comment by miina** [comment]
> I don't see how the switchEditorModeTo addition is relevant here, but it looks reasonable. 

A few times when Travis failed, the `code-editor.js` test failed and not the CTA test, apparently due to `switchEditorModeTo` using `page.click` not working reliably.

**Comment by westonruter** [comment]
Should we not milestone this for v1.2.1 instead of v1.2.2?

**Comment by miina** [comment]
Yep! Changed.

**Pull request merged**